<?xml version="1.0"?>
<launch>

  <node pkg="avc_nrc_2020" type="ACK_p-theta_step.py"
     name="p_theta_controller_node" output="screen" clear_params="true">
    <param name="delay_time" value="10"/>
    <!--<param name="points_array" value="[[10, -7], [0, 0], [-10, -7], [-10, 7], [0, 6], [10, 7], [10, -7]]"/>-->
    <param name="r" value="0.07"/>
    <param name="l" value="0.36"/>
    <param name="k_p" value="250"/>
    <param name="threshold_dist" value="2.0"/>
    <param name="target_vel" value="2.5"/>
    <param name="initial_state" value="[0, 0, 0, 0, 0, 0, 0, 0]"/>
  </node>

</launch>
